CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ include (utils.cmake)
 
 disallow_intree_builds()
 
-project (utf8proc VERSION 2.9.0 LANGUAGES C)
+project (utf8proc VERSION 2.10.0 LANGUAGES C)
 
 # This is the ABI version number, which may differ from the
 # API version number (defined in utf8proc.h and above).
```

Comment:
Please also update `SO_MINOR` to `1` (here and in the Makefile and MANIFEST).

(I don't think we need to update `SO_MAJOR`, since we only split up one undocumented field of `utf8proc_property_struct` into multiple bitfields with the same total size, so binary compatibility should be maintained.)

Reply:
I agree. Done.